spec/controllers/media_controller_spec.rb

```diff
@@ -161,81 +161,4 @@
       end
     end
   end
-
-  describe '#hash_for_auth_check' do
-    context 'cancan authorizes' do
-      before { allow(controller).to receive(:can?).and_return(true) }
-      it 'returns hash with status of success' do
```

Comment:
i don't think we have a test for success anywhere if this is removed?  Maybe keep this and a test to show we call MediaAuthenticationJSON if cannot?